smoketests/README.rst

```diff
@@ -1,5 +1,5 @@
 =================
-Systemtest README
+Smoke tests README
 =================
```

Comment:
It doesn't let me make a suggestion here for whatever reason, but the top and bottom lines need an additional `=`.

Reply:
Looks like a GitHub bug. The tool tip says "Applying suggestions on deleted lines is not supported". My guess is it's because I moved and modified the file at the same time.